recipes/mariadb-connector-cpp/all/test_package/CMakeLists.txt

```
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.16)
```

Downgrade CMake minimum version on tests to Conan's CMake requirement 3.15 
```suggestion
cmake_minimum_required(VERSION 3.15)
```